From 04a8def18caccad27292ba97dc8ea8b3eb3a6afd Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@linaro.org>
Date: Tue, 26 May 2020 11:10:45 +0200
Subject: [PATCH] scripts/pem_to_pub.py: use Cryptodome module instead of
 Crypto

Upgrade scripts/pem_to_pub.py to use module Cryptodome instead of
module Crypto for consistency with the other helper Python scripts
of OP-TEE OS package.

Signed-off-by: Etienne Carriere <etienne.carriere@linaro.org>
Acked-by: Jens Wiklander <jens.wiklander@linaro.org>
---
 scripts/pem_to_pub_c.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/pem_to_pub_c.py b/scripts/pem_to_pub_c.py
index 3a896a393..d3f0e5006 100755
--- a/scripts/pem_to_pub_c.py
+++ b/scripts/pem_to_pub_c.py
@@ -21,8 +21,8 @@ def get_args():
 
 def main():
     import array
-    from Crypto.PublicKey import RSA
-    from Crypto.Util.number import long_to_bytes
+    from Cryptodome.PublicKey import RSA
+    from Cryptodome.Util.number import long_to_bytes
 
     args = get_args()
 
-- 
2.17.1

